Backend Design - Issues #9, #10, #18, #25, #27 #26

raccoonrick opened the pull request:
Added backend functions to the pages.
Converted html files to php files
Added db folder for the necessary databases and tables
Issue #18 still in progress

raccoonrick committed:
Added Basic Login Feature (Issue #9)

Added dummy index, login and register pages along with images and css files to test the login feature. For now, admins are made by registering a username containing "admin". Will be changed in the future.  Pages will be changed alongside frontend changes.

raccoonrick committed:
Added database and modified login feature (#9)

Added the three databases for categories, users and products to allow others to simulate the project

raccoonrick committed:
Added the necessary backend files for categories and products (#18, #10 and #11)

Changed the pages so that they connect to the database and list the products from the tables with their respective details pages

raccoonrick committed:
Finished the backend for products and login (#9, #10, #11)

Also added the updated css, js and img files to the branch

J-P-engdev reviewed (approved):
Everything checks out after reviewing in meeting minutes!

raccoonrick committed:
Added Fixed Index ( #25 )

Added the updated css files and index file to the project

raccoonrick committed:
Completed the add product feature ( #18 )

Added two files, verifyproduct.php creates the necessary product page once all the information is verified.

J-P-engdev commented:
Please review so that we can merge into the main branch as requested by ricky.

superdina68 reviewed (approved):
Reviewed- no changes required
